modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyActiveClaims/index.scss

```
@@ -0,0 +1,44 @@
+.policy-active-claims-container {
+	padding: 0px 200px;
```

No need to add `px` to 0

```suggestion
	padding: 0 200px;
```

---

Done